srv/salt/ceph/packages/common/default.sls

```diff
@@ -1,11 +1,28 @@
+{% os = salt['grains.get']('os') %}
 
+{% if os is 'SUSE' %}
 
-
-stage prep dependencies:
+stage prep dependencies suse:
   pkg.installed:
     - pkgs:
       - lsscsi
       - pciutils
       - gptfdisk
     - fire_event: True
 
+{% elif os is 'Ubuntu' %}
+
+stage prep dependencies ubuntu:
+  pkg.installed:
+    - pkgs:
+      - lsscsi
+      - pciutils
+      - gdisk
```

Comment:
Googling suggests those names work for Debian too (but I haven't tested here, don't have a Debian box handy)